ci-operator/step-registry/ipi/conf/vsphere/osdisk/disktype/ipi-conf-vsphere-osdisk-disktype-ref.metadata.json

```diff
@@ -0,0 +1,15 @@
+{
+	"path": "ipi/conf/vsphere/osdisk/disktype/ipi-conf-vsphere-osdisk-disktype-ref.yaml",
```

Comment:
```suggestion
	"path": "ipi/conf/vsphere/disktype/ipi-conf-vsphere-osdisk-disktype-ref.yaml",
```

Reply:
maybe we need modify the other parameter for disk in the future, so added disktype folder under osdisk. maybe disk size or others. because i saw azure has disksize setting, and vsphere also support custom disk size.

Reply:
I guess you are coping the `osdisk` folder from azure platform, for azure, there is `osdisk` setting, but for vsphere, there is no such option.